doc/snippets/Microsoft.Data.SqlClient.Server/SqlMetaData.xml

```diff
@@ -218,7 +218,7 @@ The following are the default values assigned to `dbType`, depending on the `Sql
                 <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
-For more information, see [Table-Valued Parameters](/dotnet/framework/data/adonet/sql/table-valued-parameters).  
+For more information, see [Table-Valued Parameters](/sql/connect/ado-net/sql/table-valued-parameters).  
```

Comment:
sql folder is repeated in the path. Is it by design? It has happened in some other places.

Reply:
They were ported as it is from original location